inc/azure_c_shared_utility/tickcounter.h

```diff
@@ -5,24 +5,29 @@
 #define TICKCOUNTER_H
 
 #ifdef __cplusplus
-extern "C" {
+extern "C"
+{
 #endif /* __cplusplus */
 
 #include <stdint.h>
 
 #include "azure_c_shared_utility/umock_c_prod.h"
 
-#if _WIN32
+#ifdef __MBED__
```

Comment:
Make this an OR within the ifdef WIN32 below. You can clarify that the comment applies to WIN32 only. There is no QueryPerformanceCounter for MBED..

Reply:
fixed, and not only for QueryPerformanceCounter, a 32-bit tickcounter is going to roll over back to zero after roughly 49.7 days that is not good for IoT devices which need keep running for months